src/lang/english.txt

```diff
@@ -2347,12 +2347,11 @@ STR_FACE_FACECODE_ERR                                           :{WHITE}Couldn't
 STR_FACE_SAVE                                                   :{BLACK}Save
 STR_FACE_SAVE_TOOLTIP                                           :{BLACK}Save favourite face
 STR_FACE_SAVE_DONE                                              :{WHITE}This face will be saved as your favourite in the OpenTTD configuration file
-STR_FACE_EUROPEAN                                               :{BLACK}European
-STR_FACE_EUROPEAN_TOOLTIP                                       :{BLACK}Select European faces
-STR_FACE_AFRICAN                                                :{BLACK}African
-STR_FACE_AFRICAN_TOOLTIP                                        :{BLACK}Select African faces
 STR_FACE_YES                                                    :Yes
 STR_FACE_NO                                                     :No
+STR_FACE_SKIN_COLOUR                                            :Skin colour:
+STR_FACE_SKIN_COLOUR_TOOLTIP                                    :{BLACK}Change skin colour
+
```

Comment:
My OCD triggered: for consistency, maybe move this to just above `STR_FACE_HAIR`? The newline is now in a weird spot, compared to the other 1/2/3 settings :)